engine/include/engine/graphics/OpenGL.hpp

```diff
@@ -132,6 +133,27 @@ namespace engine::graphics {
         * @brief Clears GL_DEPTH_BUFFER_BIT, GL_COLOR_BUFFER_BIT, and GL_STENCIL_BUFFER_BIT.
         */
         static void clear_buffers();
+#
+
+
+        static unsigned int create_framebuffer();
+        static unsigned int create_color_attachment(int SCR_WIDTH, int SCR_HEIGHT);
+        static unsigned int create_render_buffer(int SCR_WIDTH, int SCR_HEIGHT);
+        static unsigned int configure_framebuffer_rectangle();
+        static void bind_framebuffer(unsigned int framebuffer);
+        static void bind_colorbuffer(unsigned int colorbuffer);
+        static void blend();
+        static void clear_color(float a, float b, float c, float d);
```

Comment:
Možda bi imena parametara mogla biti r, g, b, a, s obzirom na to da predstavljaju komponente boje? To bi moglo poboljšati čitljivost i jasnoću koda.